src/main/resources/assets/iris/lang/ru_ru.json

```diff
@@ -0,0 +1,6 @@
+{
+  "iris.shaders.reloaded": "Шейдеры перезагружены!",
+  "iris.keybind.reload": "Перезагрузить шейдеры",
+  "iris.keybinds": "Iris",
+  "iris.shaders.reloaded.failure": "Ошибка при перезагрузке шейдеров: %s"
```

Comment:
JustMarkov translated this slightly differently at https://github.com/JustMarkov/Iris/commit/d8dd925bad0a2b521beeefdd1a5155ce52dc1e9f :

```
  "iris.shaders.reloaded.failure": "Ошибка при Перезагрузке шейдеров! Причина: %s"
```

Not sure which is better, however I think JustMarkov's version is closer in form to the original English version.

However, it's possible that "Failed to reload shaders: %s" like what you did might be a better way to write things in the english version to... I'm split on this.

Reply:
I decided to change all error messages to the same as yours 
Like this "Failed to Reload Shaders! Reason: %s"